README.md

```diff
@@ -58,8 +58,204 @@ document.addEventListener('DOMContentLoaded', function() {
+        
+        <script src="/tsneez/dist/tsneez.js"></script>
+        <script src='./example3.js'></script>
+        
+        <title>tsneez Visualization</title>
+        
+        <style>
+            .viewport {
+                width: 800px;
+                height: 600px;
+            }
+            .viewport svg {
+                width: 800px;
+                height: 600px;
+                display: block;
+            }
+            .viewport svg text {
+                pointer-events: none;
+                font-family: 'Lato', Arial;
+                font-weight: 500;
+                font-size: 1em;
+                fill: #444;
+            }
+            .infoCard {
+                position: relative;
+            }
+            .infoCard .infoBody {
+                position: absolute;
+                top: 0px;
+            }
+        </style>
+        
+    </head>
+    <body>
+        <div class="container">
+            <div class="row">
+                <div class="infoCard">
+                    <div class="viewport">
+                    </div>
+                </div>
+            </div>
+        </div>
+    </body>
+</html>
+```
+
+(b) Create an example3.js javascript file in the same directory as that of the HTML document and append the following code.
+```javascript
+// Hyper parameters
+var opt = {}
+opt.theta = 0.9 // theta
+opt.perplexity = 20 // perplexity
+
+
+// define and create a tsneez instance
+var stepnum = 0
+var stepEmbedding, getEmbedding
+var tsneez = new tsneez.TSNEEZ(opt) // create a tsneez instance
+initData = function (vecs) { tsneez.initData(vecs) }
+stepEmbedding = function () { stepnum++; return tsneez.step() }
+getEmbedding = function () { return tsneez.Y }
+var visEmbedding = function(vecs, number_of_embeddings) {
+    function convertToD3Format(v) {
+        let converted = Object.assign({}, v)
+        converted.words = v.words.map(function (word) {
+            return { str: String(word), init: true,}
+        })
+        converted = {
+            words: converted.words.slice(0,number_of_embeddings),
+            vecs: converted.vecs.slice(0,number_of_embeddings),
+        }
+        return converted
+    }
+
+    function animate() {
+        stepEmbedding()
+        updateEmbedding(converted.words)
+        window.requestAnimationFrame(animate)
+    }
+
+    let converted = convertToD3Format(vecs)
+    drawEmbedding(converted)
+    window.requestAnimationFrame(animate) 
+}
+
+
+
+function dimensionReduce(vecs) {
+    var NUMBER_OF_EMBEDDINGS = 1000
+    tsneez.initData(vecs.vecs)
+    visEmbedding(vecs, NUMBER_OF_EMBEDDINGS)
+}
+
+// Fetch data from a json file.
+var shortgloveFile = '/tsneez/data/shortglove.json'
+document.addEventListener('DOMContentLoaded', function() {
+    fetch(shortgloveFile)
+        .then(data => data.json())
+        .then((vecs) => {
+            dimensionReduce(vecs)
+    });
+});
+
+// Set up visualization
+var svg
+var fadeOld = 0
```

Comment:
can remove

Reply:
We need this as these are global variables.